lib/scavenger_hunt/app/assets/stylesheets/scavenger_hunt/components/_helper.sass

```diff
@@ -9,3 +9,13 @@ a.no-border
 .end-game
   color: $text
   border-color: $text !important
+
+.game-progress
```

Comment:
This is just adding a little bit of space in between the total "time" and "completed" details on the footer. I looked for something that was somewhere near 3px in the application sass so that I could use a variable, but I couldn't find one. Is there a variable you think would be good here? @flipsasser